components/financial_assistance/config/routes.rb

```diff
@@ -12,7 +12,6 @@
     put ':step/:step', on: :member, action: 'step'
     post :step, on: :collection
     get 'step/:step', on: :member, action: 'step', as: 'go_to_step'
-    get 'help_paying_coverage', on: :collection, action: 'help_paying_coverage', as: 'help_paying_coverage'
```

Comment:
Taking out the route? Not related to this ticket.

Reply:
It many not be related to this ticket, but if we keep this, we have to add mime type for this action and cannot be testable. It is better to clean up.